pynestml/cocos/co_co_function_argument_template_types_consistent.py

```diff
@@ -73,6 +74,13 @@ def visit_simple_expression(self, node):
             return
         function_name = node.get_function_call().get_name()
         method_symbol = scope.resolve_to_symbol(function_name, SymbolKind.FUNCTION)
+
+        if method_symbol is None and ASTUtils.has_delay_variable(node.get_function_call()):
+            code, message = Messages.get_function_is_delay_variable(function_name)
+            Logger.log_message(code=code, message=message, error_position=node.get_source_position(),
+                               log_level=LoggingLevel.WARNING)
```

Comment:
This coco is run before the function call (which is actually a delayed variable) is transformed to an AST variable. Since it is not technically a function call, further checks are not required. Hence, a warning is issued and returned.

Reply:
Is it necessary to warn though, if everything checks out but we are just swapping out an ``ASTFunctionCall`` for an ``ASTVariable`` "behind the scenes"? Perhaps INFO or DEBUG logging level instead?

Reply:
Yes, you are right. There is no need for a warning. I have changed it to DEBUG.